test/chainweb/default_client_test.exs

```diff
@@ -56,19 +56,20 @@ defmodule Kadena.Chainweb.DefaultClientTest do
     end
 
     test "with an invalid body", %{url: url, header: header} do
-      {:error, [chainweb: "not enough input"]} = Client.request(url <> "/local", :post, header)
+      %Error{status: 400, title: "not enough input"} =
+        Client.request(url <> "/local", :post, header)
     end
 
     test "without header", %{url: url, body: body} do
-      {:error, [chainweb: ""]} = Client.request(url <> "/local", :post, [], body)
+      %Error{status: 400, title: ""} = Client.request(url <> "/local", :post, [], body)
```

Comment:
To avoid a lot of functions to handle each 400+ error I think we could handle it with the range and a generic message "client error", what do you think?

Reply:
@santiagoboterokommit IMO, I do prefer to handle every single error that Chainweb may return. This will make the code even more predictable.

Reply:
At least a clause to handle the most common errors: 400, 401, 404, 500+ (server error)!

Reply:
I got it, WIP